src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const Rental = 40;
+  let totalAmount = Rental * days;
```

Comment:
Add a variable like 'discountAmount' and use your variable 'totalAmount' for counting